api/src/main/scala/hmda/api/http/InstitutionsHttpApi.scala

```diff
@@ -40,7 +42,7 @@ trait InstitutionsHttpApi extends InstitutionProtocol with ApiErrorProtocol with
   val institutionsPath =
     path("institutions") {
       val path = "institutions"
-      val institutionsActor = system.actorSelection("/user/institutions")
+      val institutionsActor = system.actorSelection("/user/supervisor/institutions")
```

Comment:
Switching this and all similar calls to `system.actorSelection("/user/supervisor/institutions")` to a `FindActorByName` might be more in keeping with the established pattern of accessing persistent actors through the supervisor. It would also solve the possible issue that Nick brought up about the `InstitutionActor` itself switching off.